src/components/molecules/EventPaymentButton/EventPaymentButton.tsx

```diff
@@ -57,7 +59,7 @@ const EventPaymentButton: React.FunctionComponent<PropsType> = ({
           <button
             role="link"
             className="btn btn-primary buy-tickets-button"
-            disabled={!canUserJoinTheEvent(event)}
+            disabled={!isEventStartingSoon(event)}
```

Comment:
[NB] `useMemo`?

Reply:
Probably shouldn't need it since it's just a `boolean` field, unless the logic in `isEventStartingSoon` is particularly heavy/expensive to run.

Reply:
fixed the logic of `isEventStartingSoon`: https://github.com/sparkletown/sparkle/pull/1302/commits/ca1b2af01733aab889bb0631cda18841b82c80c2

Reply:
see comments in that other thread for an outstanding bug with it

Reply:
fixed, see the comment: https://github.com/sparkletown/sparkle/pull/1302#discussion_r634215959